package.json

```diff
@@ -16,13 +16,16 @@
   },
   "dependencies": {
     "@mantine/core": "^4.2.4",
+    "@mantine/dates": "^4.2.6",
     "@mantine/dropzone": "^4.2.5",
     "@mantine/hooks": "^4.2.4",
     "@mantine/next": "^4.2.4",
     "@mantine/notifications": "^4.2.5",
     "@mantine/prism": "^4.2.4",
     "@tabler/icons": "^1.68.0",
+    "dayjs": "^1.11.2",
```

Comment:
Where does `dayjs` get used/what is it used for?

Reply:
dayjs is a required peer dependency for @mantine/date :)